db/migrate/20220517200900_add_food_ref_inventory.rb

```diff
@@ -0,0 +1,5 @@
+class AddFoodRefInventory < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :inventories, :food
+  end
+end
```

Comment:
You don't need to create a new migration, just you have to update your model like this :

<img width="417" alt="Capture d’écran 2022-05-18 à 05 48 03" src="https://user-images.githubusercontent.com/7243132/168952675-ef951b3f-f3e6-4781-8097-7167f19379bb.png">

So in the User model you have to add : 

```
has_many :foods, dependent: :destroy
has_many :inventories, dependent: :destroy
```

In the model Inventory : 
`has_many :inventoryfoods, dependent: :destroy`

In the model Food : 
`has_many :inventoryfoods, dependent: :destroy`